City-of-Pori/pori-kada
# PORI-89 Mobile navigation changes

To test:

1. drush cc all
2. View the site on mobile device.
3. Make sure the mobile navigation hangs on for a while when you scroll down and then hides. If you scroll back up the navigation should appear again.
4. Check with local.visitpori.fi as well same things.
5. Make sure the desktop navigation works still the way it used to on both sites.
4. Extra: Check that front page main news liftups look good on mobile and the text is not attached to the sides of the browser
